maestro/scripts/sync-secret-subflows.py

```
@@ -0,0 +1,80 @@
+
+def yaml_double_quoted(value: str) -> str:
+    return '"' + value.replace("\\", "\\\\").replace('"', '\\"') + '"'
+
+
+def write_input_subflow(path: Path, secret: str, label: str) -> None:
+    path.write_text(
+        "\n".join(
+            [
+                "appId: com.android.chrome",
+                "---",
+                f"# AUTO-GENERATED by sync-secret-subflows.py — do not edit",
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)